src/dashboard/cd-dashboard-children.vue

```diff
@@ -1,32 +1,151 @@
+      <h1 class="cd-dashboard-children__header">{{ $t('My Children') }}</h1>
+      <div class="cd-dashboard-children__child" v-for="child in children">
+        <h3 class="cd-dashboard-children__name">
+          {{ child.name }}
+          <a :href="`/dashboard/profile/${child.userId}/edit`" class="cd-dashboard-children__edit-child"><i class="fa fa-pencil-square-o" aria-hidden="true"></i></a>
+        </h3>
+        <span class="cd-dashboard-children__badges" v-if="child.badges" >
+          <div class="cd-dashboard-children__badge" v-for="badge in child.badges.slice(0,2)">
+            <img class="cd-dashboard-children__badge-image" :src="badge.imageUrl" />
+            <span class="cd-dashboard-children__badge-text">{{ badge.name }}</span>
+          </div>
+          <a :href="`/dashboard/children/${child.userId}`" class="cd-dashboard-children__badges-link" v-if="child.badges.length > 2">{{ $t('See all {badgesAmount} badges', {badgesAmount: child.badges.length}) }}</a>
+        </span>
+        <p class="cd-dashboard-children__badges-none" v-else>{{ $t('{name} doesn\'t have any badges yet. Talk to the organisers of your Dojo to learn how {name} can be rewarded through badges.', { name: child.firstName }) }}</p>
+      </div>
+    </div>
+    <div v-else class="cd-dashboard-children">
+      <h1 class="cd-dashboard-children__header">{{ $t('My Children') }}</h1>
+      <div class="cd-dashboard-children__child cd-filler">
+        <h3 class="cd-dashboard-children__name cd-dashboard-children__name--filler"></h3>
+        <span class="cd-dashboard-children__badges cd-dashboard-children__badges--filler"></span>
+      </div>
     </div>
   </div>
-</div>
 </template>
 
 <script>
+  import UserService from '@/users/service';
+  import { mapGetters } from 'vuex';
+
   export default {
     name: 'cd-dashboard-children',
-    components: {
+    data() {
+      return {
+        loadedChildren: false,
+        userProfile: {},
+        userChildren: [],
+      };
+    },
+    computed: {
+      ...mapGetters(['loggedInUser']),
+      children() {
+        if (this.userChildren) {
+          return this.userChildren;
+        }
+        return null;
+      },
+      hasChildren() {
+        return (this.userChildren && this.userChildren.length > 0);
+      },
+      isDisplayable() {
+        return this.hasChildren && this.loadedChildren;
+      },
+    },
+    methods: {
+      async loadProfile() {
+        this.userProfile = (await UserService.userProfileData(this.loggedInUser.id)).body;
+      },
+      async loadChildren() {
+        if (this.userProfile.children) {
+          this.userChildren = (await Promise.all(
+            this.userProfile.children.map(
+              child => UserService.userProfileData(child))))
+            .map(res => res.body);
+        }
+      },
+    },
+    async created() {
+      await this.loadProfile();
+      this.loadChildren().then(this.loadedChildren = true);
```

Comment:
I'm not sure how this isn't throwing some sort of syntax error. I didn't think you could pass an assignment into something that's expecting a function, but I guess that's just JavaScript for you! 😅 Pretty sure this will end up assigning loadedChildren to true before children have actually loaded.

Best thing to do here is either move `this.loadedChildren = true` to the `loadChildren` function, or here in `created` do:
``` javascript
await this.loadChildren();
this.loadedChildren = true;
```